its/projects/DotCoverExcludedCoverage/dotCover.Output.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html class="main">

```

Comment:
I run the dotCover tool just like in the documentation and dumped all the files here, so that we don't need to do it on every run of the test.

Reply:
Done!